tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -39,17 +45,19 @@ def get(self, request, *args, **kwargs):
             return HttpResponseRedirect(os.environ["FRONTEND_BASE_URL"])
 
         # get the validation keys to confirm generated nonce and state
-        nonce_and_state = utils.get_nonce_and_state(request)
-        nonce_validator = nonce_and_state.get("nonce", "not_nonce")
-        state_validator = nonce_and_state.get("state", "not_state")
+        nonce_and_state = get_nonce_and_state(request.session)  # pragma: no cover
```

Comment:
are you skipping coverage on these because they are mocked in the test suite? at a minimum it seems like an integration test that hits these would be good.